### ISSUE TITLE
BUG报告范例（这里简要说明问题）

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Wlinuxhv/rt-thread#1]
```
这里详细说明BUG的细节，包括（越详细越好）：
1.如何操作会触发
2.出现BUG时的情况描述（如有Hard-Fault可贴上错误报告，也可��
�照相、截图上传）
3.如有相关附件，可点击下面“Attach a file”上传

其它字段说明：

Status 状态字段：
New - 用户提交后是这个状态
Accepted - 开发者审核后，如果确认是BUG，则修改成此状态
Started - 
相关开发者开始处理时修改成此状态（如果审核者直接处理��
�也可以直接修改到此状
态）
Fixed - 开发者处理完成后修改成此状态，等待测试
Verified - 
第三方测试确认后修改成此状态（如仍旧发现问题可追加描��
�并重新修改成
Accepted状态）
Invalid - 开发者审核后认为这个不是BUG，则修改成此状态
Duplicate - 
开发者审核后认为和另一个Issue重复，则修改成此状态
WontFix - 
开发者审核后认为这个BUG暂时不需要处理，或者已经在新版��
�中修复，则修改成此
状态
Done - 非代码修改类的问题，处理完后修改成此状态

Labels 标签字段：
Issue相关的一些属性，可以按实际情况设置，方便归类管理。

Blocked on 阻塞字段：
前置任务属性，指当前Issue必须等待另一个Issue解决后才能继�
��处理。


注意事项：
1.在提交Issue之前，先确认代码已同步到最新版，且无相同的I
ssue报告。
2.Issue是所有人可见的，不要在Issue里包含密码等敏感信息，��
�过截图等内容存在敏感信息，
请预先隐去。
```

Original issue reported on code.google.com by `Kyle.Hu.GZ@gmail.com` on 27 Feb 2010 at 3:34

**Comment by GoogleCodeExporter** [comment]
```
为保证此范例说明一直可见，请保持此Issue为开放状态。
```

Original comment by `Kyle.Hu.GZ@gmail.com` on 27 Feb 2010 at 3:37
- Changed state: **New**

**Comment by GoogleCodeExporter** [comment]
```
是否需要在内部做一个中文范本？即模板都是中文的。
```

Original comment by `bernard.xiong` on 17 Mar 2010 at 4:54
